quickwit/quickwit-serve/src/jaeger_api/rest_handler.rs

```diff
@@ -139,7 +139,7 @@ pub fn jaeger_traces_search_handler(
 #[utoipa::path(
     get,
     tag = "Jaeger",
-    path = "/otel-traces-v0_6/jaeger/api/traces/{id}",
+    path = "/otel-traces-v0_7/jaeger/api/traces/{id}",
     responses(
```

Comment:
sorry to chip in, just curious if improvements from your changes could  simplify the logic in `jaeger_get_trace_by_id` function (the part that converts `trace_id` for `GetTraceRequest`)?

Reply:
Looks like `jaeger_get_trace_by_id` is already supposed to receive its `trace_id` parameter as a hex string, so my latest changes won't likely affect the validation. Is that what you meant ?

Reply:
sorry for the confusion - my point is if it's possible to move the "String to hex" conversion of the trace_id somewhere from the `serve` crate. It's just thinking out loud not something I have strong opinion about.

Reply:
@etolbakov we can open an issue about moving jaeger specific code present in `quickwit-serve` to `quickwit-jager`